hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/error/ErrorBuilder.java

```diff
@@ -10,78 +12,204 @@
 	
 	private static String ack= "ACK";
 	
-	private static String version = "2.4\n";
+	private static String msh= "MSH";
```

Comment:
Please use consistent spacing in variable declarations. We have some instances of double spaces and some of missing spaces (e.g. before =).